.github/pull_request_template.md

```diff
@@ -13,13 +13,10 @@ _Describe the rules you have implemented and link to any relevant issues._
 
 Before submitting this PR, please make sure:
 
+- [ ] You have read the CONTRIBUTING guide in its entirety.
+- [ ] You have not used AI on any parts of this pull request.
+      (This only applies to external contributors.)
```

Comment:
```suggestion
For external contributors:
- [ ] You have read the CONTRIBUTING guide in its entirety.
- [ ] You have not used AI on any parts of this pull request.
```
this way we can just skip over these items instead of needing to click them

Reply:
I would link to the contributing guide directly.